w3d_translator/w3d_translator.py

```diff
@@ -44,42 +44,74 @@ def farewell():
     exit(0)
 
 
-# Translate a single project
+# Copy all files from source to destination
+def copy_files(source: Path, destination: Path):
+    # TODO: Catch this exception
```

Comment:
I think you can delete this. You're catching below!

Reply:
👍🏼